test/e2e/storage/utils/utils.go

```diff
@@ -191,12 +191,9 @@ func TestVolumeUnmountsFromDeletedPodWithForceOption(c clientset.Interface, f *f
 	}
 	Expect(err).NotTo(HaveOccurred())
 
-	// Wait for pod to enter "Terminating state"
-	time.Sleep(30 * time.Second)
-
 	By("Starting the kubelet and waiting for pod to delete.")
 	KubeletCommand(KStart, c, clientPod)
-	err = f.WaitForPodTerminated(clientPod.Name, "")
+	err = f.WaitForPodNotFound(clientPod.Name, time.Minute)
```

Comment:
is one minute timeout enough? Before the wairForPodTerminated waits for 5 mins, I think?

Reply:
changed to use same timeout as WaitForPodTerminated